### PR TITLE
Project: Movies by Rosanna Dahlberg & Karolin Mann

### PR DESCRIPTION
**Pull request by Rosanna86**
Link to my deployed project:
https://the-movie-pop-corner.netlify.app/

My reflections on how this week's project turned out:
Both of us really enjoyed building this website and seeing how everything we've learnt, this far, about React came together in this project.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
-    </div>
-  )
-}
+    <>
+      <BrowserRouter>
+        <Navbar title="Movie(pop)Corner" />
+        <Switch>
+          <Route exact path="/" render={() => <PopularList movies={list} />} />
+          <Route path="/details/:movieID" component={Details} />
+          <Route exact path="/toprated"> <TopRatedList /></Route>
```
Comment (Rephili) [inline code/src/App.js:30]: So nice with more categories!

File: code/src/components/BurgerMenu.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { NavLink } from 'react-router-dom';
+import "../components/BurgerMenu.css";
+
+const BurgerMenu = (props) => {
+
+    return (
+        <div className="burger">
```
Comment (Rephili) [inline code/src/components/BurgerMenu.js:8]: Looking good in the mobile version!

File: code/src/components/Details.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
+.detailsPage {
```
Comment (Rephili) [inline code/src/components/Details.css:1]: I get really inspired by your styling in general. Will definitely try to implement some similar code in my future projects. :)

File: code/src/components/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    font-family: 'Nova Square', cursive;
+    color: rgb(13, 144, 153);
+    font-size: 30px;
+    color: #fff;
+    text-align: center;
+    padding-left: 10px;
+    animation: glow 5s ease-in-out infinite alternate;
+  }
+  
+  @keyframes glow {
```
Comment (Rephili) [inline code/src/components/Navbar.css:23]: Really cool :)

File: code/src/components/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavLink } from 'react-router-dom';
+import "../components/Navbar.css"
+import BurgerMenu from './BurgerMenu';
+
+//simple navbar with props
+//by adding 'exact className' the / link isn't active all the time
```
Comment (Rephili) [inline code/src/components/Navbar.js:7]: Interesting to see how you solved this

File: code/src/components/NotFound.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
+.wrapper-404{
```
Comment (Rephili) [inline code/src/components/NotFound.css:1]: Also very well styled

File: code/src/components/Loader.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (Rephili) [inline code/src/components/Loader.js:1]: I can't see this on the page

**Review by Rephili — COMMENTED** [review]
Hi Karoline and Rosanna! You've added a lot of features to your site and it is also very well styled. The page is also structured in a clear way. Way to go! :)

File: code/src/components/Details.css (hunk 3 above)
Comment (Rosanna86) [reply code/src/components/Details.css:1]: Thank you Rebecca!

File: code/src/components/Navbar.js (hunk 5 above)
Comment (Rosanna86) [reply code/src/components/Navbar.js:7]: I have reused the same burger menu that I used for my first project and then in the WeatherApp and now here :)

File: code/src/components/Loader.js (hunk 7 above)
Comment (Rosanna86) [reply code/src/components/Loader.js:1]: It's there, I hope. We have seen it :) But maybe you can change the toggle in dev-tools to slow down the network?

**Pull request closed without merging**